modules/Bio/EnsEMBL/PipeConfig/BuildGeneMartMTMPTables_conf.pm

```diff
@@ -66,7 +66,7 @@ sub pipeline_analyses {
             -parameters      => {
                 species     => $self->o('species'),
                 antispecies => $self->o('antispecies'),
-                division    => $self->o('division'),
+                division    => [$self->o('division')],
```

Comment:
This prevents multiple divisions from being specified; could perhaps set `'division' => []` in the default_options and leave this as it was? Hopefully that'd make it a bit clearer that the parameter is to be interpreted differently.

Reply:
This was to initially copy the behavior in place in here https://github.com/Ensembl/ensembl-biomart/blob/a3d81ab14d6d6c2e87ab6c8eb837ce4b9b38aa7b/modules/Bio/EnsEMBL/PipeConfig/BuildMart_conf.pm#L120 and here https://github.com/Ensembl/ensembl-biomart/blob/a3d81ab14d6d6c2e87ab6c8eb837ce4b9b38aa7b/modules/Bio/EnsEMBL/PipeConfig/BuildMart_conf.pm#L229